clicommand/cancel_signal.go

```diff
@@ -8,31 +8,33 @@ import (
 )
 
 const (
-	defaultCancelGracePeriod = 10
+	defaultCancelGracePeriodSecs = 10
+	defaultSignalGracePeriodSecs = -1
+	defaultSignalGracePeriod     = (defaultCancelGracePeriodSecs + defaultCancelGracePeriodSecs) * time.Second
```

Comment:
```suggestion
	defaultSignalGracePeriod     = (defaultCancelGracePeriodSecs + defaultSignalGracePeriodSecs) * time.Second
```

Unless I'm misunderstanding this, we want `defaultSignalGracePeriod` to be the combination of both?

Reply:
Good catch. This has often been confusing, and I probably didn't strictly need to try to improve it.